### DISCUSSION TITLE
更新到beta3后play商店就提示这个了，beta2没有来着

### DISCUSSION THREAD

**Discussion by xfljy** [gkd-kit/gkd#647] [General]
![Screenshot_20240703_092046.jpg](https://github.com/gkd-kit/gkd/assets/134601450/b92b22f7-11c4-466d-b03b-e761b5855042)

**Comment by Ethereal49** [comment]


**Reply by xfljy** [reply]
这不很正常？之前强制要求下的，现在卸了

**Comment by lisonge** [comment]
这个是 google play 自己的策略

没有上架的使用无障碍权限的应用就会报这个警告

和反诈没有关系
